DigitalLearningSolutions.Web/Controllers/Register/RegisterController.cs

```diff
@@ -278,11 +290,25 @@ public async Task<IActionResult> Summary(SummaryViewModel model)
                         data.SupervisorDelegateId
                     );
 
+                CreateEmailVerificationHashesAndSendVerificationEmails(

```

Comment:
This doesn't feel like logic that needs to be in the controller. I think it should be merged into registrationService.CreateDelegateAccountForNewUser, and I think we can rename that method to simply "RegisterDelegateForNewUser" since this is the only place we use it, and it does really do a bit more than just creating a delegate account..